docs/source/querysets.rst

```diff
@@ -0,0 +1,33 @@
+``QuerySet`` API reference
+==========================
+
+Some MongoDB-specific ``QuerySet`` methods are available by adding a custom
+:class:`~django.db.models.Manager`, ``MongoManager``, to your model::
+
+    from django.db import models
+
+    from django_mongodb.managers import MongoManager
+
+
+    class MyModel(models.Model):
+        ...
+
+        objects = MongoManager()
+
+
+.. currentmodule:: django_mongodb.queryset.MongoQuerySet
+
+``raw_aggregate()``
+-------------------
+
+.. method:: raw_aggregate(pipeline, using=None)
+
+Similar to :meth:`QuerySet.raw()<django.db.models.query.QuerySet.raw>`, but
+instead of a raw SQL query, this method accepts a pipeline that will be passed
+to :meth:`pymongo.collection.Collection.aggregate`. The pipeline must return
+the necessary fields to construct model instances and may also return
```

Comment:
What happens if the pipeline does not return the necessary fields and what are the necessary fields? We may not need to document this but it would be nice to know what such a query looks like. Given no syntax errors, `raw_aggregate` always returned something or nothing for me so I'm not sure if it helps folks to mention this.